mlir/test/Dialect/Tensor/canonicalize.mlir

```diff
@@ -2018,3 +2018,23 @@ func.func @invalid_slice_ops(%t: tensor<?xf32>, %t2: tensor<?xf32>) -> tensor<?x
   %1 = tensor.insert_slice %0 into %t2[2][%c][1] : tensor<?xf32> into tensor<?xf32>
   return %1 : tensor<?xf32>
 }
+
+// -----
+
+// Negative dynamic dimension sizes should not be turned into static sizes.
+// They should also not prevent the op from verifying.
```

Comment:
Isn't the convention to describe test purpose in test func name and not do comments? When confused about a test, then git blame usually has all the information anyway.

Reply:
Personally, I'd be happy to see more comments in unit test cases. We have quite a few of those in the bufferization dialect unit tests.

Reply:
Fair enough.